airdrop/infrastructure/repositories/airdrop_repository.py

```diff
@@ -30,6 +30,8 @@ def get_pending_txns(self):
             pending_txns = (
                 self.session.query(ClaimHistory)
                 .filter(ClaimHistory.transaction_status == AirdropClaimStatus.PENDING.value)
+                .order_by(ClaimHistory.id.desc())
```

Comment:
on second thought lets do it aescending , the longer it is pending the more likely it will get failed

Reply:
the recent ones might get marked a Success once the event comes

Reply:
Resolved in the latest commit